docs/pages/build-reference/build-webhook.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Build webhooks
+---
+
+EAS can alert you as soon as your build has completed via a webhook. Webhooks need to be configured per-project, so if you want to be alerted about builds for both `@johndoe/awesomeApp` and `@johndoe/coolApp`, you need to run `eas webhook:create` in each directory.
+
+After running the command, you'll have to provide the webhook URL (or specify it with the `--url` flag) that handles HTTP POST requests. Additionally, you'll have to input a webhook signing secret, if you have not already provided it with the `--secret` flag. It must be at least 16 characters long, and it will be used to calculate the signature of the request body which we send as the value of the `expo-signature` HTTP header. You can use the signature to verify a webhook request is genuine (example code below). We promise that we keep your secret securely encrypted in our database.
```

Comment:
```suggestion
After running the command, you'll have to provide the webhook URL (or specify it with the `--url` flag) that handles HTTP POST requests. Additionally, you'll have to input a webhook signing secret, if you have not already provided it with the `--secret` flag. It must be at least 16 characters long, and it will be used to calculate the signature of the request body which we send as the value of the `expo-signature` HTTP header. You can use the signature to verify a webhook request is genuine (example code below).
```

"We promise" sounds a bit weird plus I'm not really sure we need to explicitly state that this value is stored securely.